ddtrace/contrib/snowflake/patch.py

```diff
@@ -14,6 +14,12 @@
     "snowflake",
     dict(
         _default_service="snowflake",
+        # FIXME: prefix snowflake span names with service
+        # The snowflake integration was introduced following a different pattern
+        # than all other dbapi-compliant integrations. It sets span names to
+        # `sql.query` whereas other dbapi-compliant integrations are set to
+        # `<service>.query`.
+        _dbapi_span_name_prefix="sql",
```

Comment:
Do we have to set this? won't we just inherit from the dbapi default if we omit?

Reply:
Put here in case we want to change in future.